picard/ui/infodialog.py

```diff
@@ -357,7 +357,7 @@ def sorttracknum(item):
             try:
                 return int(item.tracknumber)
             except:
-                return item.tracknumber
+                return item.tracknumber or 0
```

Comment:
Cant we directly return 0 here?

Reply:
Yes, I usually try to change as little code as possible, but in this case, you're right that if the cast to int failed, then we should just return 0 in any case.

Reply:
Hmmm, after thinking a bit more about it, I've made it a bit more complex and used a regular expression to extract a number if possible, since returning a string was used for tracks containing values like '3/10'. Previously, if all tracks in a cluster contained tracknumbers in such format, it worked fine since the sorting was done by comparing the strings, but if there are tracks with values '2' and '3/10', then that failed even with my commit.
I've just added another commit that parses all cases correctly and always try to convert item.tracknumber to an int.